lxd/certificates.go

```diff
@@ -164,17 +164,20 @@ func certificatesGet(d *Daemon, r *http.Request) response.Response {
 }
 
 func updateCertificateCache(d *Daemon) {
+	logger.Debug("Refreshing trusted certificate cache")
```

Comment:
Would have been nice to split the logging/cleanup bits into a separate commit for easier cherry-picking.
But I'm currently planning on cherry-picking all this work anyway, so not actually needed.